skimage/io/_io.py

```diff
@@ -49,10 +44,9 @@ def imread(fname, as_gray=False, as_grey=None, plugin=None, flatten=None,
         RGB-image MxNx3 and an RGBA-image MxNx4.
 
     """
-    if as_grey is not None:
-        as_gray = as_grey
-        warn('`as_grey` has been deprecated in favor of `as_gray`'
-             ' and will be removed in v0.16.')
+    if 'as_grey' in plugin_args.keys()
```

Comment:
Missing colon here :). Also, for dicts you can use `if k in d` instead of `if k in d.keys()`.